lib/rubygems/ext/ext_conf_builder.rb

```diff
@@ -39,35 +31,22 @@ def self.build(extension, dest_path, results, args=[], lib_dir=nil, extension_di
 
       ENV["DESTDIR"] = nil
 
-      make dest_path, results, extension_dir, tmp_dest_relative
-
-      full_tmp_dest = File.join(extension_dir, tmp_dest_relative)
+      rel_dest_path = Pathname.new(dest_path).relative_path_from(Pathname.new(extension_dir))
```

Comment:
I prefer the `Pathname(dest_path)` syntax, without `.new`, it's cleaner and easier to read.

Reply:
I actually find the more explicit `Pathname.new` easier to pase, I guess because it's more common Ruby. And it's also the style we generally use in the rest of the codebase. That said, I was also considering to not introduce a dependency on `pathname` just for this, and instead modify the existing `get_relative_path` helper. Will figure this out later.